Docs/source/running_cpp/parameters.rst

```diff
@@ -1042,6 +1042,12 @@ Numerics and algorithms
 
     Note that in the current implementation in WarpX these 3 numbers must be equal.
 
+* ``interpolation.galerkin_scheme`` (`0` or `1`)
+    Whether to use a Galerkin scheme when gathering fields to particles.
+    When set to `1`, the interpolation order for each field component is reduced by one along the direction of that component.
+    For example, `E_z` is gathered using ``interpolation.nox``, ``interpolation.noy``, and ``interpolation.noz - 1``.
+    Defaults to `1` unless ``warpx.do_nodal = 1`` and ``algo.field_gathering = energy-conserving``.
```

Comment:
Looking at the code: should this be a non-exclusive "or"?
```suggestion
    Defaults to `1` unless ``warpx.do_nodal = 1`` or ``algo.field_gathering = energy-conserving``.
```

Reply:
I agree.
Also, shouldn't this be ` and/or ``algo.field_gathering = momentum-conserving```  instead of `energy-conserving`?

Reply:
Yes, should be fixed now